CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 This is only used for recording changes for major version bumps.
 More minor changes may optionally be recorded here too.
 
+## 57.8.0
```

Comment:
Was this meant to be `58.0.0` to match version.py?

Reply:
oops yes, typo. fixed.